src/schema/null.ts

```diff
@@ -38,6 +39,6 @@ export function transformNullsToUndefined(
   return graphqlArgs
 }
 
-function isObject(obj: any): boolean {
-  return obj && typeof obj === 'object'
+function isObject(arg: any): arg is Record<string, any> {
```

Comment:
Let's just bring lodash into this and use its `isObject`. This here still doesn't account for `null` for example. And I don't feel like having to remember why that's ok this time.

Don't mind the extra package size at all. Once we tree shake instead of module shake, we'll save cost anyways. But 500kb pales in comparison to the weight of other deps here and product time/user disruption in getting these details right.

Reply:
> This here still doesn't account for null for example

It does though. First condition is `arg &&`. Not sure there's a reason to bring it now ?